README.md

```diff
@@ -124,7 +124,7 @@ This will be included when app is done.
 
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_ in line 152 of your ReadMe, as this project is done solely. This will make your ReadMe appear more professional.